Minesweeper.py

```diff
@@ -36,7 +36,7 @@ def print_stats():
 def update_player_pos(direction):
     x, y = player_pos
     if direction == "up":
-        y -= 1
+        y -= 2  # Error: should be 1
```

Comment:
The calculation for moving "up" is incorrect. It should be adjusted to decrease `y` by 1 instead of 2.
```diff
-        y -= 2  # Error: should be 1
+        y -= 1  # Corrected to 1
```
---